### PR TITLE
Adding basic test for API

### PR DESCRIPTION
**Pull request by heaven00**
_Add details of your PR here.._

## Checklist
- [x] Have you followed the [guidelines for contributing](https://github.com/javatarz/github-contribution-leaderboard/blob/master/contributing.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/javatarz/github-contribution-leaderboard/pulls) for the same feature you're adding?
- [x] Have you run tests (`pipenv run pytest`) before sending this PR?
- [x] Have you run linting (`./lint.sh`) before sending this PR?

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -26,3 +29,14 @@ def data():
 @app.route('/<path:path>')
 def send_js(path):
     return send_from_directory('web/src', path)
+
+
+@app.route('/')
+def index():
+    return redirect(url_for('data'))
+
```
Comment (heaven00) [inline app.py:37]: @javatarz what should be the response for base URL?

Comment (javatarz) [reply app.py]: the `/` route should probably redirect to  `/index.html` because that renders the web page. Sounds like a good smart default to have :)

Comment (javatarz) [reply app.py:37]: the `/` route should probably redirect to `/index.html` because that renders the web page. Sounds like a good smart default to have :)

**Review by javatarz — CHANGES_REQUESTED** [review]

**Comment by javatarz** [comment]
Had a word with @heaven00. We're closing this because of the code that @rahulbagad is writing.

**Pull request closed without merging**